ugs-platform/ugs-platform-ugscore/src/main/java/com/willwinder/ugs/nbp/core/toolbars/Bundle.properties

```diff
@@ -0,0 +1,4 @@
+mainWindow.swing.firmware.toolbarTitle = Firmware selection toolbar
```

Comment:
Unfortunatly I cant use the localizing service for the action display name (the annotation requires a static final variable). I was hoping that it could be assigned dynamically in the constructor and it will work everywhere, except for in the toolbar customization... :(
```java
@ActionRegistration(
        iconBase = PauseAction.ICON_BASE,
        displayName = "<Not localized> PauseAction",
        lazy = false)

    ...

    public PauseAction() {
        ...
        putValue(NAME, LocalizingService.PauseTitle);
        putValue("displayName", LocalizingService.PauseTitle);
    }
```
![screen shot 2017-04-17 at 20 01 58](https://cloud.githubusercontent.com/assets/8962024/25098847/5ce885ce-23aa-11e7-9c35-338a013957ec.png)

By giving a resource bundle it will work if the bundle exist in the same module with the key. But this leads us to the same problem with multiple bundle files. 
```java
@ActionRegistration(
        iconBase = PauseAction.ICON_BASE,
        displayName = "resources.MessagesBundle#mainWindow.swing.pauseButton",
        lazy = false)
```

I'll move the definitions to the ´LocalizingService.java´. If we want to use the correct display name for the actions I think we need to consider having local bundles in the package with a default translation. All overrides can still be done in the central MessagesBundle. But this is for you to decide. :)